_data/simple-icons.json

```diff
@@ -1248,6 +1248,11 @@
                 "url": "https://github.com/jashkenas/backbone/blob/master/LICENSE"
             }
         },
+        {
+            "title": "Backendless",
+            "hex": "3A72C1",
```

Comment:
I'm not sure about the color. I think it's either `#FFFFFF` (the fill color of the logo on the website) or `#5056E5` (the accent color used on the website). But I can't find `#3A72C1` anywhere. I tend to prefer `#FFFFFF` because that's the color used in the logo, so it feels more associated with the brand IMO.

Reply:
Sounds good! I'll make the change